### ISSUE TITLE
Address insufficient contrast accessibility findings in cloud.gov Pages application

### ISSUE THREAD

**Issue by svenaas** [cloud-gov/pages-core#4132]
This task documents findings from cloud-gov/product#2496

Many links have insufficient visual contrast amounting to a WCAG failure under AA and/or AAA:

`/sites`

Number varies.

* “Add Site” link (white on blue)
* All “View Repo” and “View Site” links (blue on grey)
* “Return to top” link (blue on white)

`sites/id/builds`

Number varies. 232 contrast errors, for example, for 18F

* “View Repo” (blue on white)
* “View Site” (blue on white)
* “Auto Refresh:” and “OFF” (blue on white)
* “Refresh builds” (blue on white)
* Branch name, preview, view logs (blue on white)
 
`sites/id/builds/build_id/logs`

3 contrast errors:

* “View Repo” (blue on white)
* “View Site” (blue on white)
* “Download logs” (white on blue)

`/sites/id/published`

Number varies. 356 contrast errors, for example, for 18F

* “View Repo” (blue on white)
* “View Site” (blue on white)
* All “Preview site” and “View Files” (blue on white)

`sites/id/published/branch_name`

Number varies. A spot-checked site on staging, for example, has 203 contrast errors.

* “View Repo” (blue on white)
* “View Site” (blue on white)
* All “View” links (blue on white)
* "Previous" and "Next" buttons (white on blue)

`sites/id/settings`

Number varies. 5 contrast errors, for example, for 18F

* “View Repo” (blue on white)
* “View Site” (blue on white)
* “these settings” (blue on white)
* “viewing site previews” (blue on white)
* “live URL” (blue on white)

`/organizations`

2 contrast errors:

* “Edit” (blue on grey)
* “Return to top” (blue on white)

`/organizations/id`

1 contrast error:

* “Return to top” (blue on white)

`/settings`

1 contrast error:

* “Connect with Github” (white on blue)

`/sites/new`

2 contrast errors:

* “View sample” link (blue on grey)
* “Use this template” link (white on blue)

## Summary:

These contrast errors are likely addressable with a small number of theme changes — something like:
* Dominant link color (blue on white needs a darker blue)
* Occasional link color (blue on grey needs a darker blue, or possibly a lighter grey, though it's already pretty light)
* Button link color (white on blue needs a darker blue)

## Acceptance criteria:

WAVE scans of the URLs above should find zero contrast errors for WCAG AA or AAA for either large or normal with the presented contrast.

**Comment by svenaas** [comment]
I have repeated the WAVE scan against dev with the changes proposed in #4134 and found zero contrast errors.

**Comment by svenaas** [comment]
Fixed with #4134, which is now up on staging.
